docs/entity/components.md

```diff
@@ -526,6 +526,54 @@ See the following example:
 }
 ```
 
+## `Index`
+
+A field used to specify [index](https://docs.splunk.com/Documentation/Splunk/latest/Admin/Inputsconf#Scripted_Input:).
```

Comment:
This link goes to scripted inputs, maybe a link to the Splexicon would be more accurate?

Reply:
Can you please update a link?